src/dstack/_internal/server/services/fleets.py

```diff
@@ -179,17 +179,37 @@ async def list_project_fleet_models(
     return list(res.unique().scalars().all())
 
 
-async def get_fleet_by_name(
-    session: AsyncSession, project: ProjectModel, name: str
+async def get_fleet(
+    session: AsyncSession, project: ProjectModel, name: Optional[str], id: Optional[uuid.UUID]
```

Comment:
`id` local variable shadows a Python's built-in. Alternatives: `fleet_id` or `id_`